portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,19 +70,57 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the comment from the form.
-    String content = request.getParameter("title");
-    long timestamp = System.currentTimeMillis();
-    String ipAddr = request.getRemoteAddr();
+    final String content = request.getParameter("message");
+    final long timestamp = System.currentTimeMillis();
+    final String ipAddr = request.getRemoteAddr();
+
+    // Get the image from the form.
+    final String imageUrl = getUploadedFileUrl(request, "image");
```

Comment:
what should happen if this is null?

Reply:
If it's null, it will just be stored as null in the entity. Should I be adding something else to this?

Reply:
Maybe we should not store any imageUrl if there is no such value.

Production application would need to pay for storage, so they are likely to wish to reduce data footprint.

Reply:
Ah right that makes sense. Will change it now.

Reply:
Done. Added an 'if' case to only store the image URL when it is given.